src/BenchmarkDotNet/Running/BenchmarkRunnerClean.cs

```diff
@@ -84,6 +85,27 @@ internal static Summary[] Run(BenchmarkRunInfo[] benchmarkRunInfos)
 
                     foreach (var benchmarkRunInfo in supportedBenchmarks) // we run them in the old order now using the new build artifacts
                     {
+                        if (benchmarkRunInfo.Config.Options.IsSet(ConfigOptions.Resume))
+                        {
+                            var directoryInfo = new DirectoryInfo(rootArtifactsFolderPath);
+                            var lastFiles = directoryInfo.GetFiles($"{title.Split('-')[0]}*");
+                            if (lastFiles.Count() > 1)
+                            {
+                                var lastFilesSorted = lastFiles.OrderBy(o => o.LastWriteTime).ToArray();
+                                var lastUpdatedFile = lastFilesSorted[lastFiles.Length - 2];
```

Comment:
`lastFiles.Length - 2` is wrong, as it's not last, but one before last.

```suggestion
                                var lastUpdatedFile = lastFiles.OrderByDescending(file => file.LastWriteTime).First();
```

Reply:
> `lastFiles.Length - 2` is wrong, as it's not last, but one before last.

I know that. The **last is the current file** and we need one before the last where we can get the last benchmarkId.

Reply:
Sure @adamsitnik I'll do it now, thanks.